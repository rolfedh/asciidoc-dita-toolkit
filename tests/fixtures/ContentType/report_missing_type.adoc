// A topic without the content type definition:
= Topic title

A paragraph.

