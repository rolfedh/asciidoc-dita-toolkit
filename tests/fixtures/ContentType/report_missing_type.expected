:_mod-docs-content-type: PROCEDURE
// A topic without the content type definition:
= Topic title

A paragraph.
